osidb/tests/test_flaw.py

```diff
@@ -1075,34 +1077,34 @@ def test_validate_impact_and_summary(self, impact, summary, should_alert, alert)
 
         if should_alert:
             assert flaw.alerts.count() == 1
-            assert flaw.alerts.filter(name="impact_without_summary").exists()
+            assert flaw.alerts.filter(name=alert).exists()
```

Comment:
I wonder how this worked before? :smile:

Reply:
The only alert used on the test was `impact_without_summary` that's why it worked 😆, but the `alert` variable was declared